File: 백준/Bronze/2562. 최댓값/최댓값.java
import java.util.Scanner;

public class Main {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		int array1[] = new int[9];
		int max = 0;
		int index = 0;
		
		for(int i=0;i<array1.length;i++) {
			array1[i] = sc.nextInt();
			
			if(array1[i]>max) { 
				max = array1[i]; 
				index = i+1; 
			}
		}
		sc.close();
		
		System.out.println(max);;
		System.out.println(index);
		
	}

}

File: 백준/Bronze/2562. 최댓값/최댓값.cs
int[] array1 = new int[9];
int max = 0, index = 0;

for (int i = 0; i < array1.Length; i++)
{
    array1[i] = int.Parse(Console.ReadLine());

    if (array1[i] > max)
    {
        max = array1[i];
        index = i + 1;
    }
}

Console.WriteLine(max);
Console.WriteLine(index);